2_correlation_analysis/spell_vs_counts_experiment/README.md

```diff
@@ -0,0 +1,23 @@
+# Compare SPELL vs counts correlation
+
+There are two possible correlation matrices we can use for our analysis:
+
+1. Correlation of the MR counts expression matrix
+2. Correlation of the MR counts that have been processed using SPELL
```

Comment:
```suggestion
1. Correlation of the median-ratio counts expression matrix
2. Correlation of the median-ratio counts that have been processed using SPELL
```
I'd suggest spelling this out here, in case people haven't read your other scripts and documentation.